commands_to_load/Administration.py

```diff
@@ -9,7 +9,7 @@
 class Administration():
 
 	async def botManager(self, ctx):
-		for managers in self.admin['BOT_MANAGERS']:
+		for managers in self.admin_list['BOT_MANAGERS']:

```

Comment:
why rename it?

Reply:
@wnfldchen

Reply:
it was broken before, it was "red underlined" error unresolved reference

Reply:
(`self.admin` doesnt exist, `self.admin_list` does)

Reply:
huh, i missed that. thanks for alerting me to it